.gitignore

```diff
@@ -50,4 +50,6 @@ Session.vim
 .cargo
 !/src/test/run-make/thumb-none-qemu/example/.cargo
 no_llvm_build
+**node_modules
+**package-lock.json
```

Comment:
I don't think this is valid syntax. At least I can't find it in `man gitignore` and ripgrep complains about it:
```
rust/.gitignore: line 55: error parsing glob '**node_modules': invalid use of **; must be one path component
rust/.gitignore: line 56: error parsing glob '**package-lock.json': invalid use of **; must be one path component
```

Reply:
`**` is to look for all subfolders, whatever the level.

Reply:
I believe that would be `**/node_modules`

Reply:
It works for me as is but don't hesitate to send an update.